parm/metplus_config/stats/cam/grid2obs/PointStat_fcstHREFmean_obsPREPBUFR_SFC.conf

```diff
@@ -37,47 +37,40 @@ PROCESS_LIST = PointStat
 BOTH_VAR1_NAME = TMP
 BOTH_VAR1_LEVELS = Z2
 
-BOTH_VAR2_NAME = UGRD
+BOTH_VAR2_NAME = WIND
 BOTH_VAR2_LEVELS = Z10
+BOTH_VAR2_THRESH =  >=10.3, >=15.4, >=20.6
 
-BOTH_VAR3_NAME = VGRD
-BOTH_VAR3_LEVELS = Z10
+BOTH_VAR3_NAME = DPT
+#OBS_VAR3_NAME = TDO
+BOTH_VAR3_LEVELS = Z2
 
-BOTH_VAR4_NAME = WIND
-BOTH_VAR4_LEVELS = Z10
-BOTH_VAR4_THRESH =  >=10.3, >=15.4, >=20.6
 
-BOTH_VAR5_NAME = DPT
-#OBS_VAR5_NAME = TDO
-BOTH_VAR5_LEVELS = Z2
+BOTH_VAR4_NAME = HGT
+BOTH_VAR4_LEVELS = L0
+BOTH_VAR4_THRESH = <152, <305, <914, <1524, <3048, >=914
+FCST_VAR4_OPTIONS = GRIB_lvl_typ = 215
 
+BOTH_VAR5_NAME = CAPE
+BOTH_VAR5_LEVELS = L0
+BOTH_VAR5_THRESH = >=250, >=500, >=1000, >=2000
+FCST_VAR5_OPTIONS = GRIB_lvl_typ = 1
 
-BOTH_VAR6_NAME = HGT
+BOTH_VAR6_NAME = VIS
 BOTH_VAR6_LEVELS = L0
-BOTH_VAR6_THRESH = <152, <305, <914, <1524, <3048, >=914
-FCST_VAR6_OPTIONS = GRIB_lvl_typ = 215
-
-BOTH_VAR7_NAME = CAPE
-BOTH_VAR7_LEVELS = L0
+BOTH_VAR6_THRESH = <805, <1609, <4828, <8045, <16090, >=8045
+
+FCST_VAR7_NAME = CAPE
+FCST_VAR7_LEVELS = P90-0
+FCST_VAR7_OPTIONS = cnt_thresh = [ >0 ]
+OBS_VAR7_NAME = MLCAPE
+OBS_VAR7_LEVELS = L0-100000
+OBS_VAR7_OPTIONS = cnt_thresh = [ >0 ]; cnt_logic = UNION
 BOTH_VAR7_THRESH = >=250, >=500, >=1000, >=2000
-FCST_VAR7_OPTIONS = GRIB_lvl_typ = 1
 
-BOTH_VAR8_NAME = VIS
+BOTH_VAR8_NAME = TCDC
 BOTH_VAR8_LEVELS = L0
-BOTH_VAR8_THRESH = <805, <1609, <4828, <8045, <16090, >=8045
-
-FCST_VAR9_NAME = CAPE
-FCST_VAR9_LEVELS = P90-0
-FCST_VAR9_OPTIONS = cnt_thresh = [ >0 ]
-OBS_VAR9_NAME = MLCAPE
-OBS_VAR9_LEVELS = L0-100000
-OBS_VAR9_OPTIONS = cnt_thresh = [ >0 ]; cnt_logic = UNION
-BOTH_VAR9_THRESH = >=250, >=500, >=1000, >=2000
-
-BOTH_VAR10_NAME = TCDC
-BOTH_VAR10_LEVELS = L0
-BOTH_VAR10_THRESH = <10, >=10, >=50, >=90
-
+BOTH_VAR8_THRESH = <10, >=10, >=50, >=90
 
```

Comment:
@BinbinZhou-NOAA Can you explain all the changes in this particular file?  Seems more numerous than all the other files.  You had said that you had other warnings not in Wei's doc, but it looks like you're changing several variables here.